lewis/adapters/epics.py

```diff
@@ -101,6 +125,10 @@ def process_pv_updates(self, force=False):
             if self._timers[pv] >= pv_object.poll_interval or force:
                 try:
                     self.setParam(pv, getattr(self._target, pv_object.property))
+
+                    if pv_object.meta_data_property is not None:
+                        self.setParamInfo(pv, getattr(self._target, pv_object.meta_data_property))
```

Comment:
Unlike `setParam`, `setParamInfo` calls `updatePVs` every time it is called. Since it's called in a loop here, that means we could get multiple `updatePVs` calls in a row in a single cycle.

I'm not sure if this is an issue in any way... just something to keep an eye on.